workitem/table_join.go

```diff
@@ -43,8 +43,8 @@ type TableJoin struct {
 	// object to be activated.
 	PrefixActivators []string // e.g. []string{"iteration."}
 
-	// disallowedColumns specified all fields that are allowed to be queried
-	// from the foreign table. When empty all columns are allowed.
+	// AllowedColumns specified all fields that are allowed to be queried from
+	// the foreign table. When empty all columns are allowed.
```

Comment:
Can you add a comma, please, after the `empty`? 
Like `When empty, all columns are allowed`. I couldn't understand the sentence at first. :)

Reply:
sure

Reply:
Done in 6000fdb